examples/api/async_api.py

```diff
@@ -0,0 +1,24 @@
+import asyncio
+
+from dotenv import load_dotenv
+
+from hatchet_sdk import Hatchet, WorkflowList
+
+load_dotenv()
+
+hatchet = Hatchet(debug=True)
+
+
+async def main():
+    hatchet_async_rest_client = await hatchet.get_async_rest_client()
```

Comment:
i think we have hatchet.admin and hatchet.aio.admin can you look into that pattern and can we do the same for this and instantiate the aio client on first get maybe?

Reply:
also it looks like we're replacing the sync api? is this a breaking change?

Reply:
not a breaking change unless someone is doing something fucky with the library that involves touching the underlying objects on the `RestApi` object.


I completely forgot about `hatchet.aio`, let me see if this'll work. The main issue is that `aiohttp` requires an active event loop or it explodes so layering it on our existing hatchet client was looking pretty terrible and so I opted for the current design.